api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -129,13 +128,13 @@ public void setUp() {
 
     registeredTier = TestMockFactory.createRegisteredTierForTests(accessTierDao);
 
-    DbUser user = new DbUser();
-    user.setUsername(USERNAME);
-    user = userDao.save(user);
-    dbUser = user;
+    dbUser = new DbUser();
```

Comment:
not sure this is actually any different, but Hibernate paranoia (like the comment for UserServiceImpl) has served me well in the past

Reply:
This seems more logical to me and one less variable declaration.

Reply:
Thanks.  This will survive in a different PR 👍